torchinductor/overrides.py

```diff
@@ -36,6 +39,70 @@ def replace_fx(gm: torch.fx.GraphModule):
     gm.recompile()
     return gm
 
+class EltwiseFusionOp:
+    def __init__(self, post_op, scalars=[], algorithm=""):
+        self.post_op = post_op
+        self.scalars = scalars
+        self.algorithm = algorithm
+
+class LinearEltwise(nn.Linear):
+    def __init__(self, linear, eltwise, op_name, op_info, in_features, out_features, bias, device, dtype):
+        super(LinearEltwise, self).__init__(in_features, out_features, bias=bias,
+            device=device, dtype=dtype)
+        self._update_module_params(linear, eltwise, op_name, op_info)
+
+    def _update_module_params(self, linear, eltwise, op_name, op_info):
+        self.__dict__ = copy.deepcopy(linear.__dict__)
+
+        self.attr = op_name
+
+        assert all(hasattr(eltwise, item) for item in op_info.scalars)
+        self.scalars = [getattr(eltwise, item) for item in op_info.scalars]
+
+        algorithm = ""
+        if op_info.algorithm:
+            assert hasattr(eltwise, op_info.algorithm)
+            algorithm = getattr(eltwise, op_info.algorithm) 
+        self.algorithm = algorithm
+
+    def forward(self, input):
+        y = torch.ops.mkldnn_prepacked.linear_eltwise(input, self.weight, self.bias, self.attr, self.scalars, self.algorithm)
+        return y
+
+def fuse_linear_eltwise_eval(linear, eltwise, op_name, op_info):
+    return LinearEltwise(linear,
+                    eltwise,
+                    op_name,
+                    op_info,
+                    linear.in_features,
+                    linear.out_features,
+                    linear.bias is not None,
+                    linear.weight.device,
+                    linear.weight.dtype)
+
+def fuse_fx(gm: torch.fx.GraphModule, example_inputs):
```

Comment:
Not sure if it is proper to add backend-specific FX passes here. Would it better to add a FX pass registration mechanism here (e.g., a registration decorator similar to how lowerings are registered) and move the backend-specific passes to a separate backend file?